Gulpfile.js.BASE

```diff
@@ -0,0 +1,39 @@
+const gulp = require('gulp');

```

Comment:
files with `.BASE`, `.LOCAL`, `.REMOTE`, `.orig` should not be commited ever.

Reply:
Reverted that commit.